superflore/generators/bitbake/run.py

```diff
@@ -81,9 +81,8 @@ def main():
 
     # clone current repo
     selected_targets = active_distros
-    # try:
-    link_existing_files(args.ros_distro)
-    """
+    try:
+        link_existing_files(args.ros_distro)
```

Comment:
Related do we really need to symlink? Why don't we simple change the working directory/pass the repository location as an argument?